install_config/configuring_pipeline_execution.adoc

```diff
@@ -0,0 +1,50 @@
+[[install-config-configuring-pipeline-execution]]
+= Configuring Pipeline Execution
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+== Overview
+
+{product-title} utilizes Jenkins to execute continuous development pipelines.  To
+provide a seamless experience for users, the first time a user creates a build
+configuration using the xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[pipeline]
+strategy type, a set of Jenkins related resources will be created within the user's project.
+By default, the system will look for a template named `*jenkins*` in the `*openshift*` namespace and
+instantiate it within the user's project.  {product-title} ships with a default template definition
+which will, upon instantiation, create the following in the user's project:
+
+* a deployment configuration for Jenkins using the official {product-title} Jenkins image
+* a service and route for accessing the Jenkins deployment
```

Comment:
service name will be "jenkins", users need to make sure they don't have "jenkins" service already, otherwise the pipeline jenkins will not be instantiated.